config/locales/client.en.yml

```diff
@@ -2748,23 +2745,7 @@ en:
         remove_timer: "remove timer"
 
       actions:
-        flag: "Flag"
-        defer_flags:
-          one: "Ignore flag"
-          other: "Ignore flags"
-        undo:
-          off_topic: "Undo flag"
-          spam: "Undo flag"
-          inappropriate: "Undo flag"
-          bookmark: "Undo bookmark"
-          like: "Undo like"
         people:
-          off_topic: "flagged this as off-topic"
-          spam: "flagged this as spam"
-          inappropriate: "flagged this as inappropriate"
-          notify_moderators: "notified moderators"
-          notify_user: "sent a message"
-          bookmark: "bookmarked this"
```

Comment:
I couldn't find any usage of those strings in our code base, so I think they aren't used anymore since the introduction of the review queue. Only the `post.actions.undo.*` strings are used in a test script, but I'm not familiar with it. @eviltrout Any tips on how to fix it?

https://github.com/discourse/discourse/blob/7df57b35dae86007eb75b040ea76b7809f14733f/script/test_hbs_compiler.rb#L5

Reply:
That test script can be safely ignored.